packages/firebase_ml/lib/src/firebase_ml.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of firebase_ml;
+
+/// The Firebase machine learning API.
+///
+/// You can get an instance by calling [FirebaseML.instance]
+class FirebaseML {
+  FirebaseML._();
+
+  /// Singleton of [FirebaseML].
+  static final FirebaseML instance = FirebaseML._();
+
+  static const MethodChannel _channel =
+      const MethodChannel('plugins.flutter.io/firebase_ml');
+
+  /// Example call across channel
+  static Future<String> get doSomething async {
+    final String version = await _channel.invokeMethod('downloadRemoteModel');
+    return version;
+  }
```

Comment:
This call is to ensure that app compiles. Basically while I'm refactoring, adding and removing stuff I check that I don't forget anything or remove anything needed.

I will probably use it a lot when working on download to differentiate between when I broke the system and when only that specific call is broken. I would like to keep it for now simply because it does the simplest job.

Reply:
Resolving